admin-ui/plugins/admin/redux/audit/Resources.js

```diff
@@ -2,3 +2,4 @@ export const SCRIPT = 'custom-script'
 export const API_ROLE = 'api-role'
 export const API_PERMISSION = 'api-permission'
 export const API_MAPPING = 'api-role-mapping'
+export const API_USERS = 'api-users'
```

Comment:
why `API_USERS ` here in `admin` plugin.

Reply:
removed from admin plugin